### ISSUE TITLE
AttributeError: module 'torch.compiler' has no attribute 'is_compiling'

### ISSUE THREAD

**Issue by LukeLIN-web** [NVIDIA/apex#1857]
**Describe the Bug**

**Minimal Steps/Code to Reproduce the Bug**
 

.conda/envs/opensora/lib/python3.9/site-packages/apex/normalization/fused_layer_norm.py", line 412, in forward
    if torch.jit.is_tracing() or torch.jit.is_scripting() or torch.compiler.is_compiling() or not input.is_cuda:
AttributeError: module 'torch.compiler' has no attribute 'is_compiling'

`pip install -v --disable-pip-version-check --no-cache-dir --no-build-isolation --config-settings "--build-option=--cpp_ext" --config-settings "--build-option=--cuda_ext" git+https://github.com/NVIDIA/apex.git` I installed 
torch                     2.2.2

**Environment**

apex                      0.1  
Versions of relevant libraries:
[pip3] galore-torch==1.0
[pip3] numpy==1.26.4
[pip3] rotary-embedding-torch==0.5.3
[pip3] torch==2.2.2
[pip3] torchvision==0.17.2
[pip3] triton==2.2.0
[conda] galore-torch              1.0                      pypi_0    pypi
[conda] numpy                     1.26.4                   pypi_0    pypi
[conda] rotary-embedding-torch    0.5.3                    pypi_0    pypi
[conda] torch                     2.2.2                    pypi_0    pypi
[conda] torchvision               0.17.2                   pypi_0    pypi
[conda] triton                    2.2.0                    pypi_0    pypi

**Comment by venom8898** [comment]
I am also running in to this. I also had to had to upgrade diffusers to 0.31.0 to even get the models to download. But now when I try their multigpu example I run in to the "AttributeError: module 'torch.compiler' has no attribute 'is_compiling'" error

**Comment by kkahatapitiya** [comment]
Instead of using the master, but rather checking-out a previous tag (https://github.com/NVIDIA/apex/tree/24.04.01) before building, solved this for me.

**Comment by zyang6** [comment]
> Instead of using the master, checking-out a previous tag (https://github.com/NVIDIA/apex/tree/24.04.01) before building, solved this for me.
Have you solved this? Are you compiling apex packages before 24.01.01?

**Comment by kkahatapitiya** [comment]
Yes, using the version with tag 24.01.01 (`git checkout tags/24.01.01`), and building from source solved this issue for me.

**Comment by zyang6** [comment]
> Yes, using the version with tag 24.01.01 (`git checkout tags/24.01.01`), and building from source solved this issue for me.

Thank you !I'll give it a try

**Comment by AlongWY** [comment]
try this one 试试这个？[apex wheels](https://github.com/AlongWY/apex_wheels/releases/tag/v24.4.1)

**Comment by yinghuozijin** [comment]
> Yes, using the version with tag 24.01.01 (`git checkout tags/24.01.01`), and building from source solved this issue for me.

Correct the typo: **24.04.01**
